src/client/Microsoft.Identity.Client/Internal/Requests/SilentRequestHelper.cs

```diff
@@ -0,0 +1,44 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System.Collections.Generic;
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.Identity.Client.Cache.Items;
+using Microsoft.Identity.Client.Internal.Requests;
+using Microsoft.Identity.Client.OAuth2;
+
+namespace Microsoft.Identity.Client.Internal
+{
+    internal static class SilentRequestHelper
```

Comment:
how is this different from the one in the CacheSilentStrategy class?

Reply:
This is a helper class I added to not introduce duplicate code. Since OBO refresh flow required these 2 methods which are similar to the silent flow.